lib/console/hotspot_stats/hotspot_stats.ex

```diff
@@ -39,7 +39,7 @@ defmodule Console.HotspotStats do
         GROUP BY hotspot_address
       ) stats
       LEFT JOIN hotspots h ON stats.hotspot_address = h.address
-      LEFT JOIN organization_hotspots os ON stats.hotspot_address = os.hotspot_address
+      LEFT JOIN organization_hotspots os ON stats.hotspot_address = os.hotspot_address and os.organization_id = $1
```

Comment:
aliases of the same hotspot from other orgs is getting into the current org's table